### ISSUE TITLE
Accuracy not the same as shown in the Post

### ISSUE THREAD

**Issue by gkrish19** [kratzert/finetune_alexnet_with_tensorflow#53]
Hello,

I tried to use the same code as posted for the AlexNet network and then used the validation for the same. I have tried the same code as in the post and am getting a very bad accuracy for some reason. 

('Class name:', 'zebra', ' and Probability ', 0.6496317)
('Class name:', 'sea lion', ' and Probability ', 0.34226722)
('Class name:', 'llama', ' and Probability ', 0.36589968)

I used python and not Jupyter notebook for the validation so I printed the accuracy in the command line. I'm not sure why I have a different value. Can anyone give me some on this?

**Comment by kratzert** [comment]
I'm not sure if I understand you right. You should get the same results whether you run the code in a jupyter notebook or in a python script. If not, you must have changed something or not copied everything. But it is hard to tell from your description what you have done (wrong)

**Comment by yangerkun** [comment]
@gkrish19 I got the same bad results.

**Comment by gkrish19** [comment]
@kratzert So I ran the code as per the one given in the post without any change.  Here I have used Python 2.7 and not Jupyter. I don't know why the results are not good. Is it because it's not python3.

**Comment by kratzert** [comment]
Maybe. I don't know and won't test it. It is explicitly stated, that this code is for Python 3 so you should try this. If you get the same wrong results with python 3 let me know

**Comment by beebrain** [comment]
I got the same result. In Python 3.5 and tensorflow  #1.5, I had a result same @gkrish19 when running with bvlc_alexnet.npy weight on the Jupyter.

**Comment by TotalVariation** [comment]
I got the same result. Then I found the issue is the setting of parameters:
lrn(2, 2e-05, 0.75, name='norm1')，lrn(2, 2e-05, 0.75, name='norm2')
refer to http://www.cs.toronto.edu/~guerzhoy/tf_alexnet/myalexnet_forward_newtf.py

**Comment by beebrain** [comment]
@TotalVariation  Thanks man. I changed the parameters and executed it. I have a good result.
This problem is solved by @TotalVariation

**Comment by kratzert** [comment]
Sorry for not answering for such a long time.I became father recently and found not much time to look into this. Seems like there were some mistakes and not updated images due to some adaptions of the normalization factor. I did now update the factors according to the link posted by @TotalVariation and updated the image in the post as well with the correct probabilities after running the notebook again.
